SQL/New_patches/2018-09-07_EEG_tables.sql

```diff
@@ -0,0 +1,259 @@
+-- Create a physiological_modality table
+CREATE TABLE `physiological_modality` (
+  `PhysiologicalModalityID` TINYINT UNSIGNED NOT NULL AUTO_INCREMENT,
+  `PhysiologicalModality`   VARCHAR(50)      NOT NULL,
+  PRIMARY KEY (`PhysiologicalModalityID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+
+-- Create a physiological_output_type table
+CREATE TABLE `physiological_output_type` (
+  `PhysiologicalOutputTypeID` INT(5) UNSIGNED NOT NULL AUTO_INCREMENT,
```

Comment:
There seems to be a lot of inconsistencies with the precision of the primary keys.. why are some int(5), some tinyint, some int(10), etc?

Reply:
Normally for PK, it is preferable no to limit to much the size on the attributes. People can add/remove/add a lot, specially during testing phase. If auto_increment is not reseted,  you migth risk reaching the limit.
If you plan to have about 20~25 values, should not be a problem. But my great friend Murphy tend to show up at the least expected occasion. How about INT(4), that would leave 65K possibilities.

Reply:
@PapillonMcGill Perfect! Will change it to INT(4) then

Reply:
Done in the next commit!!